tendrl/commons/commands/config_manager.py

```diff
@@ -0,0 +1,83 @@
+#! /usr/bin/env python
+
+from etcd import Client as etcd_client
+from jinja2 import Environment
+from jinja2 import FileSystemLoader
+import json
+import os
+import platform
```

Comment:
Ack.. Also please suggest where this correctly fits commons or node-agent.
I am getting confused because https://github.com/Tendrl/commons/pull/661/files/d4b7b0d652528b213a427db53b043652408c5950#diff-059917ac9f73bcf3a89b77a6b4806857R29 is the place where its used.. But all this goes from node-agent itself and every other collectd related piece is also in node-agent